### ISSUE TITLE
Labels not reachable by keyboard navigations

### ISSUE THREAD

**Issue by EmmaDawsonDev** [EddieHubCommunity/good-first-issue-finder#105]
### Description

The labels under the search bar are not reachable by tabbing through the site. The tab jumps directly from search to the first issue.

To do: Make labels tabbable and give them focus which matches the rest of the site.

**Comment by saiswathit** [comment]
Hello Can I work on this issue
